tests/test_2022_03_24.py

```diff
@@ -0,0 +1,234 @@
+    ),
+    EVMScriptFactoryAdded(
+        factory_addr='0x54058ee0E0c87Ad813C002262cD75B98A7F59218',
+        permissions=('0xB9E5CBB9CA5b0d659238807E84D0176930753d86' + 'f6364846')
+    )
+]
+
+hyperelliptic_rockx_comp = Payout(
+    token_addr=lido_dao_token,
+    from_addr=dao_agent_address,
+    to_addr=hyperelliptic_rockx_multisig,
+    amount=350_000 * (10 ** 18)
+)
+
+def test_2022_03_24(
+    helpers, accounts, ldo_holder, dao_voting, ldo_token,
+    vote_id_from_env, bypass_events_decoding, easy_track
+):
+    dao_balance_before = ldo_token.balanceOf(dao_agent_address)
+    hr_multisig_balance_before = ldo_token.balanceOf(hyperelliptic_rockx_multisig)
+    rewards_manager_balance_before = ldo_token.balanceOf(one_inch_reward_manager)
+
+    # if rewards manager has no tokens transfer it from ldo_holder
+    if rewards_manager_balance_before == 0:
+        ldo_token.transfer(one_inch_reward_manager, 1 * 10 ** 18, {"from": ldo_holder})
+    rewards_manager_balance_before = ldo_token.balanceOf(one_inch_reward_manager)
+    assert rewards_manager_balance_before > 0
+
+    factories_before = easy_track.getEVMScriptFactories()
+
+    for f in factories_to_add_with_vote:
+        assert f.factory_addr not in factories_before
+
+    ##
+    ## START VOTE
+    ##
+    vote_id = vote_id_from_env or start_vote({'from': ldo_holder}, silent=True)[0]
+
+    tx: TransactionReceipt = helpers.execute_vote(
+        vote_id=vote_id, accounts=accounts, dao_voting=dao_voting
+    )
+
+    factories_after = easy_track.getEVMScriptFactories()
+
+    rewards_manager_balance_after = ldo_token.balanceOf(one_inch_reward_manager)
+    hr_multisig_balance_after = ldo_token.balanceOf(hyperelliptic_rockx_multisig)
+    dao_balance_after = ldo_token.balanceOf(dao_agent_address)
+
+    assert hr_multisig_balance_after - hr_multisig_balance_before == hyperelliptic_rockx_comp.amount
+    assert rewards_manager_balance_after == 0
+
+    assert dao_balance_before - dao_balance_after == \
+        hyperelliptic_rockx_comp.amount + (rewards_manager_balance_after - rewards_manager_balance_before)
+
+    for f in factories_to_add_with_vote:
+        assert f.factory_addr in factories_after
+
+    _check_factories_sanity(accounts[5], accounts[6], ldo_token, easy_track)
+
+    ### validate vote events
+    # assert count_vote_items_by_events(tx) == 6, "Incorrect voting items count"
+
+    display_voting_events(tx)
+
+    if bypass_events_decoding:
+        return
+
+    evs = group_voting_events(tx)
+
+    # asserts on vote item 1
+    validate_evmscript_factory_added_event(evs[0], factories_to_add_with_vote[0])
+
+    # asserts on vote item 2
+    validate_evmscript_factory_added_event(evs[1], factories_to_add_with_vote[1])
+
+    # asserts on vote item 3
+    validate_evmscript_factory_added_event(evs[2], factories_to_add_with_vote[2])
+
+    # asserts on vote item 4
+    validate_payout_event(evs[3], hyperelliptic_rockx_comp)
```

Comment:
I have an unexpected failure at this line with the error **"brownie.exceptions.EventLookupError: Unknown key 'to' - the 'Transfer' event includes these keys: sender, receiver, value"** from the `validate_payout_event` method.

Reply:
Seems strange, payout events validation has been widely used before by me and teammates. :thinking:

Reply:
Looks like you have a different ABI for the LDO contract (i.e. event fields are same but names differ)

Reply:
Looks so. I don't know why but in my case, the Transfer event has the wrong param names. And I can't find these names in abi files.
<img width="486" alt="image" src="https://user-images.githubusercontent.com/14151334/159810595-5fd5a709-8f77-4d74-9e4b-8b476c9aad04.png">

Reply:
I don't think it's a blocker for us. But it might be an interesting case to check why it happens.